### ISSUE TITLE
[Improvement] Use glob pattern to search .txt and .lab files in train and test directory

### ISSUE THREAD

**Issue by apsdehal** [inspirehep/magpie#92]
It would be good if the searching for `.txt` and `.lab` in train directory used glob patterns like `./**/*.txt`. This would allow structuring files inside the train directory for the user as he wants. I can work on this if you want this improvement included.

**Comment by jstypka** [comment]
Yeah, I think it would be cool. I think Python has a `glob` module that we could use for that, hopefully it should work for both Python 2 and 3.

**Comment by apsdehal** [comment]
Sure, I will start working on this within two weeks.
